docs/guide/data_types/timeseries.rst

```diff
@@ -275,16 +169,15 @@ use the sunpy plot as the foundation for a more complicated figure:
    # Modify the figure here
    fig.savefig('figure.png')
 
+.. _manipulating-timeseries:
 
-5 Manipulating TimeSeries
-=========================
+4. Manipulating TimeSeries
+==========================
 
-5.1 Modifying the Data
-======================
+4.1 Modifying the Data
+----------------------
 
-Since the timeseries data is stored as a Pandas `~pandas.DataFrame`
-you can easily modify the data directly using all of the usual Pandas methods:
-for example, you can modify a single cells value using: ::
+Since the timeseries data is stored as a Pandas `~pandas.DataFrame` you can easily modify the data directly using all of the usual Pandas methods: for example, you can modify a single cells value using: ::
```

Comment:
```suggestion
Since the timeseries data is stored as a `~pandas.DataFrame` you can use all of the usual Pandas methods.
For example, you can modify a single cells value using:

.. codeblock:: python
```